BraintreeCore/src/main/java/com/braintreepayments/api/AnalyticsDatabase.kt

```diff
@@ -1,11 +1,19 @@
 package com.braintreepayments.api
 
 import android.content.Context
+import androidx.room.AutoMigration
 import androidx.room.Database
 import androidx.room.Room
 import androidx.room.RoomDatabase
 
-@Database(entities = [AnalyticsEvent::class], version = 1)
+// Ref: https://developer.android.com/training/data-storage/room/migrating-db-versions
+@Database(
+        version = 2,
+        entities = [AnalyticsEvent::class],
+        autoMigrations = [
+            AutoMigration(from = 1, to = 2)
+        ]
+)
```

Comment:
I am trying to figure out why we need this in Android. We use FPTIBatchData in iOS.